src/utils/data-manager.js

```diff
@@ -707,40 +778,63 @@ export default class DataManager {
     }
   }
 
-  sortList(list) {
-    let columnDef = this.columns.find((_) => _.tableData.id === this.orderBy);
-    if (!columnDef) {
-      columnDef = this.columns[0];
-    }
-    let result = list;
+  sortBy(list, columnsDefs, orderCollection) {
+    const sort = this.sort;
+    const getFieldValue = this.getFieldValue;
```

Comment:
Why is this extracted?

Reply:
I thought it was going to be reused, but I thinks it not, let me revert it back to the sort function.